engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/analyse/AliasAnalysis.scala

```diff
@@ -575,8 +575,13 @@ case object AliasAnalysis extends IRPass {
             name.name
           )
         if (!nameOccursInScope) {
+          val argScope = if (susp) {
```

Comment:
I know how you love my style comments so... this really asks for ternary operator.

Reply:
There are no ternary operators in Scala! One has to use `if then else`!

Reply:
Or are you asking for a one-liner? 2ed8104